### DISCUSSION TITLE
Select Dependent Input & Passing an Repeater to a Resource

### DISCUSSION THREAD

**Discussion by 1Chrollo3** [filamentphp/filament#12709] [Help] [answered]
Hello, new in using filament,  can someone help me on how to change a specific textinput/select when a parent dropdown changes its value. Also, how do you pass the value of a repeater through a create resource? so i can access the values using foreach. Thanks.

**Comment by leandrocfe** [comment, marked as answer]
https://filamentphp.com/docs/3.x/forms/advanced#dependant-select-options
https://filamentphp.com/docs/3.x/forms/fields/repeater#using-get-to-access-parent-field-values

**Comment by 1Chrollo3** [comment]
Thank you Sir!
